tests/testsuite/old_cargos.rs

```diff
@@ -591,6 +611,11 @@ foo v0.1.0 [..]
 #[cargo_test]
 #[ignore]
 fn avoids_split_debuginfo_collision() {
+    // Test needs two different toolchains.
+    // If the default toolchain is stable, then it won't work.
```

Comment:
There's a non-breaking space here.

```suggestion
    // If the default toolchain is stable, then it won't work.
```